sequencer/src/lib.rs

```diff
@@ -206,6 +205,14 @@ pub mod network {
 )]
 pub struct Node<N: network::Type>(PhantomData<fn(&N)>);
 
+// Using derivative to derive Clone triggers the clippy lint
```

Comment:
Added this because I wasn't able to successfully disable the clippy lint inline.